business/iter8.go

```diff
@@ -145,6 +148,37 @@ func (in *Iter8Service) CreateIter8Experiment(namespace string, body []byte) (mo
 	return iter8ExperimentDetail, nil
 }
 
+func (in *Iter8Service) UpdateIter8Experiment(namespace string, name string, body []byte) (models.Iter8ExperimentDetail, error) {
+	var err error
+	promtimer := internalmetrics.GetGoFunctionMetric("business", "Iter8Service", "CreateIter8Experiment")
```

Comment:
CreateIter8Experiment -> UpdateIter8Experiment to collect metrics for this method